e2e/cypress/integration/account_settings/display/account_settings_position_spec.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @account_setting
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('Account Settings > General > Position', () => {
+    let testTeam;
+    let testUser;
+    const position = 'Master hacker';
+
+    beforeEach(() => {
+        cy.apiAdminLogin();
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+
+            cy.apiLogin(testUser);
+
+            // # Visit town square
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+        });
+    });
+
+    it('MM-T2063 Position', () => {
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').should('be.visible').click();
+
+        // # Click on Account settings menu item
+        cy.findByText('Account Settings').should('be.visible').click();
+
+        // # Fill-in the position field
+        cy.findByText('Position').should('be.visible').click();
```

Comment:
both the edit button and the header have an id which might be a better option to get the element.

this text-vs-id also happens in other tests.

Reply:
Yes I was confused about how to choose text over id? I guess in this case I used it because of readability, as well as in the context of the modal there is no conflicts with another element with a similar text

Reply:
`id` is more on implementation details but we prefer the use of text whenever possible since it's resembles on how the "real" user "finds" it.